### PR TITLE
Two more commands (time and date)

### PR DESCRIPTION
**Pull request by nicosommi**
He @jsynowiec
I use your extension (thanks btw) and recently I was just inserting the date and copying just the time here and just the date there. So basically I realized that I needed two different masks and commands.
This PR adds two commands: insert date and insert time. And also adds two configs for those commands with different defaults.
Also adds docs for them and a utility json that I used to run and test the changes by pressing F5.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": ["${workspaceRoot}/out/**/*.js"],
-			"preLaunchTask": "compile"
+			"preLaunchTask": "npm"
```
Comment (jsynowiec) [inline .vscode/launch.json]: Please revert back to `compile` task.

File: README.md [hunk 2]
```diff
@@ -28,16 +28,22 @@ Open [Command Palette](https://code.visualstudio.com/docs/editor/codebasics) by
 Following commands are available:
 
 * `Insert DateTime` (<kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>I</kbd> on OS X, <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> on Windows and Linux) - Inserts current date and/or time according to configured format at the cursor position, 
+* `Insert Date` - Inserts current date according to configured format (formatDate) at the cursor position,
```
Comment (jsynowiec) [inline README.md]: Please code-quote the formats - e.g. `formatDate`

**Review by jsynowiec — CHANGES_REQUESTED** [review]
Thanks for contribution. It requires a few minor changes to be merged - I've left you comments.

**Comment by nicosommi** [comment]
Right on.
With the new tasks.json the compile task works, so it is reverted.
Also code quoted those words.

**Review by jsynowiec — APPROVED** [review]

**Pull request merged**